tests/test_stack.py

```diff
@@ -129,5 +130,19 @@ def test_LinkedListStack(self):
 
         self.assertTrue(stack.is_empty())
 
+class TestOrderedStack(unittest.TestCase):
+    def test_OrderedStack(self):
+        stack = OrderedStack()
+        self.assertTrue(stack.isEmpty())
+        stack.push(1)
+        stack.push(4)
+        stack.push(3)
+        "bottom - > 1 3 4 6 "
+        self.assertEqual([1, 3, 4, 6], stack.push(6))
```

Comment:
@danghai Shouldn't `stack.push(item)` return `None`. We can't test for it to be equal to the list.

Reply:
Yes, the unittest was made by one of the collaborators for the previous code. I have removed return statement. Will modify the unittest